content/en/docs/zero-code/java/quarkus.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Quarkus instrumentation
+linkTitle: Quarkus
+---
+
+## Quarkus OpenTelemetry instrumentation
+
+[Quarkus](https://quarkus.io/) is an open source framework designed to help
+software developers build efficient cloud native applications both with JVM and
+Quarkus native image applications.
+
+Quarkus uses extensions to provide optimized support for a wide range of
+libraries. The
+[Quarkus OpenTelemetry extension](https://quarkus.io/guides/opentelemetry)
+provides:
+
+- Out of the box instrumentation
+- OpenTelemetry SDK autoconfiguration, supporting almost all system properties
+  defined for the [OpenTelemetry SDK](/docs/languages/java/configuration/)
+- [Vert.x](https://vertx.io/) based OTLP exporter
+- The same instrumentations can be used with native image applications, which
+  are not supported by the OpenTelemetry Java agent.
+
+> [!NOTE] Quarkus OpenTelemetry instrumentation is maintained and supported by
+> Quarkus. For details, please check the
+> [Quarkus community support page](https://quarkus.io/support/).
+
+Quarkus can also be instrumented with the [OpenTelemetry Java agent](../agent/)
+if you are not running a native image application.
+
+## Getting started
+
+To enable OpenTelemetry in your Quarkus application, add the
+`quarkus-opentelemetry` extension dependency to your project.
+
+{{< tabpane >}} {{< tab header="Maven (`pom.xml`)" lang=Maven >}}
```

Comment:
Display of the tabbed content is invalid:

> <img width="444" alt="image" src="https://github.com/user-attachments/assets/ec78e735-d040-42c8-aae6-22f4bc795349" />

You need to ensure that the content gets processed as markdown, like this:

```suggestion
{{< tabpane >}} {{% tab header="Maven (`pom.xml`)" lang=Maven %}}
```